openmdao/core/total_jac.py

```diff
@@ -1242,7 +1242,8 @@ def compute_totals(self):
                             varlist = '(' + ', '.join([name for name in par_deriv[key]]) + ')'
                             print('Solving color:', key, varlist)
                         else:
-                            print('Solving variable:', key)
+                            print('Solving variable: {0} \nIn mode: {1} \n'
```

Comment:
Did you run the tests? I suspect this should make tests fail in test_pyoptsparse_driver and test_scipy_optimizer.

If they don't fail, then you should add a test for this.

Reply:
Yes, I did run the tests but I'll write one for it

Reply:
added